### PR TITLE
Minor stability adjustments in tests

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy [hunk 1]
```diff
@@ -36,9 +36,33 @@ class GrpcBaseSpecification extends HealthCheckSpecification {
...
+        def v2Numbers = v2Matcher.collect { it[0] }
+        for (int i = 0; i < v1Numbers.size(); i++) {
+            if (v1Numbers[i] > v2Numbers[i]) {
+                return 1
+            } else if (v1Numbers[i] < v2Numbers[i]) {
+                return -1
+            }
+        }
+        return 0
+    }
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy:66]: You can use spaceship(`v1 <=> v2`) operator

Comment (rtretyak) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy:66]: to achieve what result? v1 and v2 are strings, I don't need to compare strings, I need to compare versions.
`"500.2.12" <=> "500.2.7" == -1`
but ``compareNoviVersions("500.2.12", "500.2.7") == 1``

File: src-java/projectfloodlight/openflowj/build.gradle [hunk 2]
```diff
@@ -46,9 +46,9 @@ task generateSources(type: Exec, dependsOn: cloneLoxigenRepo) {
         commandLine 'make', 'java'
     }
     else {
-        def uid = new UnixSystem().getUid()
-        def gid = new UnixSystem().getGid()
-        commandLine 'docker', 'run', '--rm', '-v', "$workingDir:$workingDir", '-w', "$workingDir", '-u', "$uid:$gid", 'kilda/base-ubuntu', 'make', 'java'
+//        def uid = new UnixSystem().getUid()
```
Comment (andriidovhan) [inline src-java/projectfloodlight/openflowj/build.gradle:49]: what's wrong with that?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/GrpcBaseSpecification.groovy:66]: agree, this operator doesn't work well with string

File: src-java/projectfloodlight/openflowj/build.gradle (hunk 2 above)
Comment (rtretyak) [reply src-java/projectfloodlight/openflowj/build.gradle:49]: ops, accidentally committed from my windows machine -_-
reverted changes in this file

**Pull request merged**